control/veda_deliver_3play.py

```diff
@@ -94,6 +101,40 @@ def validate_media_url(self):
                 )
             )
 
+    def get_available_languages(self):
+        """
+        Gets available languages
+        """
+        response = requests.get(url=build_url(self.base_url, self.available_languages_url, apikey=self.api_key))
+        if not response.ok:
+            raise ThreePlayMediaLanguagesRetrievalError(
+                'Error while retrieving available languages: {response} -- {status}'.format(
```

Comment:
Don't forget to add logger prefix for better tracking.

Reply:
This is just an exception message. Logger prefix is where we are catching it.